extensions/ql-vscode/src/vscode-tests/no-workspace/interface-utils.test.ts

```diff
@@ -73,20 +72,17 @@ describe('interface-utils', () => {
   describe('resolveWholeFileLocation', () => {
     it('should resolve a whole file location', () => {
       const mockDatabaseItem: DatabaseItem = ({
-        resolveSourceFile: sinon.stub().returns(vscode.Uri.parse('abc')),
+        resolveSourceFile: sinon.stub().returns(vscode.Uri.file('abc')),
       } as unknown) as DatabaseItem;
       expect(
         tryResolveLocation(
-          {
-            t: LocationStyle.WholeFile,
-            file: 'hucairz',
-          },
+          'file://hucairz:1:1:1:1',
           mockDatabaseItem
         )
       ).to.deep.equal(
         new vscode.Location(
-          vscode.Uri.parse('abc'),
-          new vscode.Range(0, 0, 0, 0)
+          vscode.Uri.file('abc'),
+          new vscode.Range(0, 0, 0, 1)
```

Comment:
I'm slightly suspicious at the test output changing. We're sure this is ok? Was it wrong before?